src/sentry/static/sentry/app/views/alerts/rules/details/relatedTransactions.tsx

```diff
@@ -186,9 +186,9 @@ class RelatedTransactions extends React.Component<Props> {
         'project',
         'p95()',
         'count_unique(user)',
-        `user_misery(${organization.apdexThreshold})`,
+        `count_miserable(user, ${organization.apdexThreshold})`,
       ],
-      orderby: `user_misery(${organization.apdexThreshold})`,
+      orderby: `count_miserable(user, ${organization.apdexThreshold})`,
```

Comment:
The "miserable users" column on the rules/details page is still called "user misery". Should I rename it it "miserable users" so we don't have different meanings for user misery across the product?

Reply:
To add some more context, "user misery" nor pretty consistently refers to the computed score in performance and discover and not the actual number of unique miserable users.

Reply:
Hmmm I feel like here on related transactions workflow was trying to be consistent with performance, so we should stick with `user_misery` rather than flipping it to `count_miserable`

Though (and can be in a future PR/backlog) we should allow `count_miserable` alerting later

Reply:
Okay, pulling in user misery here will now show the user misery bars

Reply:
jk it was showing the bars previously as well, so no difference here other than better calibration